mis_builder/static/src/components/mis_report_widget.css

```diff
@@ -80,8 +80,14 @@
     padding-right: 1px;
 }
 
-.footnote_mis_builder_list {
+.footnote_mis_builder_table {
```

Comment:
I had not noticed this, but let's prefix all css classes with `.oe_mis_builder` ?